hpaction/forms.py

```diff
@@ -1,5 +1,48 @@
 from django import forms
 
+from project.forms import YesNoRadiosField
+from .models import FeeWaiverDetails
+
+
+class FeeWaiverIncomeForm(forms.ModelForm):
+    class Meta:
+        model = FeeWaiverDetails
+        fields = [
+            'income_amount_monthly',
+            'income_src_employment',
+            'income_src_hra',
+            'income_src_child_support',
+            'income_src_alimony',
+            'income_src_other',
+        ]
+
+
+class FeeWaiverExpensesForm(forms.ModelForm):
+    class Meta:
+        model = FeeWaiverDetails
+        fields = [
+            'rent_amount',
+            'expense_utilities',
+            'expense_cable',
+            'expense_phone',
+            'expense_childcare',
+            'expense_other',
+        ]
+
+
+class FeeWaiverMiscForm(forms.ModelForm):
+    class Meta:
+        model = FeeWaiverDetails
+        fields = ['asked_before']
+
+    asked_before = YesNoRadiosField()
+
+
+class FeeWaiverForm(forms.ModelForm):
```

Comment:
It _really_ sucks that this form only needs to exist so we can use its `resolve` static method.  We should have a better way of doing this.

Reply:
Fixed in 3421bb7!